packages/reactotron-server/package.json

```diff
@@ -4,12 +4,12 @@
   "description": "A reactotron server",
   "main": "index.js",
   "scripts": {
-    "compile": "tsc",
+    "compile": "tsc -p tsconfig.server.json",
```

Comment:
If we are building everything with the same config we can possibly just rename the config to be the default one

Reply:
Good call.

Reply:
#817 fixes this.  Thx.